docker_files/recipes/python/meta/run.yaml

```diff
@@ -0,0 +1,8 @@
+runtime: native
+config_set:
+  shell:
+    bootcmd: /python
+    env:
+      PYTHONHOME: /pyenv
```

Comment:
I suggest we do it like this:
```yaml
bootcmd: /python $ARGS
env:
  PYTHONHOME: /pyenv
  ARGS: 
```
So that user will be able to provide path to script like this:
```
$ capstan run demo --boot python --env ARGS=/mysript.py
```